node/src/Commands.ts

```diff
@@ -1974,6 +1974,61 @@ export function createXTrim(
     return createCommand(RequestType.XTrim, args);
 }
 
+export type StreamRangeBound =
+    /**
+     * Stream ID boundary used to specify the minimum stream entry ID. Can be used in the `XRANGE` or `XREVRANGE` commands
+     * to get the first stream ID.
+     */
+    | "-"
+    /**
+     * Stream ID boundary used to specify the maximum stream entry ID. Can be used in the `XRANGE` or `XREVRANGE` commands
+     * to get the last stream ID.
+     */
+    | "+"
+    /**
+     * Stream ID boundary used to specify a range of IDs to search. Stream ID bounds can be complete with
+     * a timestamp and sequence number separated by a dash ("-"), for example "1526985054069-0". Stream ID bounds can also
+     * be incomplete, with just a timestamp. Can be specified as inclusive or exclusive, where inclusive is the default.
+     */
+    | {
+          exclusive?: "(";
```

Comment:
why not make this similar to ScoreBoundary with [a boolean isInclusive that defaults](https://github.com/valkey-io/valkey-glide/blob/main/node/src/Commands.ts#L1536) to false.